build/images/Makefile

```diff
@@ -10,34 +10,27 @@ SHELL=/usr/bin/env bash -euo pipefail
 # Get the process ID of the process running Make
 MAKEPID:= $(shell echo $$PPID)
 
-all: index.txt chartmap
+all: index.txt chartmap.csv
 
-index.txt: $(manifests_images) docker/index.txt
+index.txt: docker/index.txt $(manifests_images)
 	cat $^ | sort -u | parallel -j 75% --retries 5 --halt-on-error soon,fail=100% ./inspect.sh | sort -u > $@
 
 # Kill (SIGINT) the make PGID to short-circuit parallel makes (fail fast)
-manifests/%.txt: ../../manifests/%.yaml get_base pull_images
+manifests/%.txt: ../../manifests/%.yaml
 	@mkdir -p $(@D)
 	./extract.sh $< | sort -u > $@ || kill -INT -$(MAKEPID)
 
 docker/index.txt: ../../docker/index.yaml
 	@mkdir -p $(@D)
-	../../hack/list-images.py $< | sort -u > $@
-
-get_base:
 	./get_base.sh
-
-pull_images:
```

Comment:
Having targets named `get_base` and `pull_images` actually made `Makefile` non-idempotent, because files named `get_base` and `pull_images` do not exist, so these targets (and all heir dependencies) are always executed. Moved call to these scripts to `docker/index.txt` target (to call them only once).